### PR TITLE
Recreate the EGL surface after a resize in order to correctly handle …

### PR DESCRIPTION
**Pull request by jason-simmons**
…screen rotation on Android

### CODE REVIEW

**Comment by jason-simmons** [comment]
@chinmaygarde

File: sky/shell/platform/android/platform_view_android.cc [hunk 1]
```diff
@@ -129,6 +129,30 @@ class AndroidGLContext {
 
   bool SwapBuffers() { return eglSwapBuffers(display_, surface_); }
 
+  SkISize GetSize() {
+    EGLint width = 0;
+    EGLint height = 0;
+    if (!eglQuerySurface(display_, surface_, EGL_WIDTH, &width) ||
```
Comment (chinmaygarde) [inline sky/shell/platform/android/platform_view_android.cc]: Are EGLBooleans, 0 and 1? Not quite sure.

**Comment by chinmaygarde** [comment]
lgtm. Thanks!

File: sky/shell/platform/android/platform_view_android.cc (hunk 1 above)
Comment (jason-simmons) [inline sky/shell/platform/android/platform_view_android.cc]: yes - see the "Scalar Types" section in the EGL spec

**Pull request merged**